printer_mods/Slidr/PanzerBalls2.4/readme.md

```diff
@@ -0,0 +1,23 @@
+What are Panzerballs?
+- Panzerballs are another type of feet using commonly available squash balls.
+
+Why do they exist?
+- i hate sourcing of rubber compressor feet
+- i felt that other squash ball mods do not satisfy me visually (sry)
+
+Why do i need Panzerballs?
+- You probably don´t need them unless you got problems with sourcing the normal rubber compressor feet 
+or you want something more squishy with maybe a better noise reduction.
+
+How do i install Panzerballs?
+- Panzerballs are a drop in replacement for the base plate
+- You need:
+  - 4x squash balls (super slow, double yellow dot, diameter 39.5-40.5 mm)
+    o german source:https:\\www.decathlon.de\p\squashballe-sb-990-2er-pack-zwei-gelbe-punkte\_\R-p-312192
```

Comment:
Please replace all `\` with `/` in the URL, it is invalid to use `\` in a URL.  It should be

    https://www.decathlon.de/p/squashballe-sb-990-2er-pack-zwei-gelbe-punkte/_/R-p-312192